docs/projects.rst

```diff
@@ -31,30 +31,47 @@ We recommend that projects have roughly the following folder and file structure:
 | └ Dockerfile / docker-compose.yaml
 |
 
-The ``client`` folder is commonly referred to as the *compute package*. The file ``fedn.yaml`` is the FEDn Project File. It contains information about the ``entry points``. The entry points are used by the client to compute model updates (local training) and local validations (optional) . 
-To run a project in FEDn, the client folder is compressed as a .tgz bundle and pushed to the FEDn controller. FEDn then manages the distribution of the compute package to each client. 
-Upon recipt of the package, a client will unpack it and stage it locally.
+The ``client`` folder is commonly referred to as the *compute package* and it contains files with logic specific to one client. The file ``fedn.yaml`` is the FEDn Project File and contains information about the fedn commands which runs files in the compute package. These fedn commmands are referred to as ``entry points`` and there are up to four entry points in the project folder example given above that need to be specified: 
```

Comment:
Good clarifications, but can you try to update the wording a bit? "..fedn commands.." -> "..the commands that fedn will run when a client receives requests..." ? Something like that.